frontend/app_flowy/packages/flowy_editor/example/lib/plugin/image_node_widget.dart

```diff
@@ -26,7 +29,37 @@ class _ImageNodeWidget extends StatelessWidget {
     required this.editorState,
   }) : super(key: key);
 
-  String get src => node.attributes['image_src'] as String;
+  @override
+  State<_ImageNodeWidget> createState() => __ImageNodeWidgetState();
+}
+
+class __ImageNodeWidgetState extends State<_ImageNodeWidget> with Selectable {
+  Node get node => widget.node;
+  EditorState get editorState => widget.editorState;
+  String get src => widget.node.attributes['image_src'] as String;
+
+  @override
+  List<Rect> getSelectionRectsInSelection(Offset start, Offset end) {
+    final renderBox = context.findRenderObject() as RenderBox;
```

Comment:
What if the `RenderObject` is not a `RenderBox`?

Reply:
Depends on the user's computation. It must be a `RenderBox` in this situation.

Reply:
Documenting it in your code would be better.